packages/pathfinder/src/pathfinder.ts

```diff
@@ -76,28 +76,61 @@ export class Pathfinder {
     }
 
     /**
+     *  TODO: Use or remove
+    // TODO: should this be moved to stellar section
      * Uses Stellar SDK to retrieve and verify the balances for REFERENCE_ACCOUNT_PUBLIC_KEY 
      */
     private getStellarBalances() {
         return server.loadAccount(Pathfinder.REFERENCE_ACCOUNT_PUBLIC_KEY)
             .then((account: any) => account.balances)
     }
 
-    private makeKey = (asset: any) => {
+    // todo: is this okay to be static
+    // TODO: should this be moved to stellar section
+    private static makeCompositeKey = (asset: any): string => {
         const { asset_code, asset_issuer, asset_type} = asset;
-        const key = (asset_type !== 'native') ?
-            `${asset_code}-${asset_issuer}`
+        const key = (asset_type !== 'native') 
+            ? `${asset_code}-${asset_issuer}`
             : 'XLM';
         return key;
     }
 
+    private static makeCompositeKeyFromPath = (pathResult: any): string => {
```

Comment:
Needs function description. If you must have two different functions for making composite keys, you should reuse the logic.